### ISSUE TITLE
About code "-out3.detach() + out3 (line 35 in the birealnet.py)"

### ISSUE THREAD

**Issue by hadhe145** [liuzechun/ReActNet#22]
Thanks for the nice paper and code! The work is really impressive.

But I have a little confusion when I read the code "out = out_forward.detach() - out3.detach() + out3" (line 35 in the birealnet.py) and "binary_weights = binary_weights_no_grad.detach() - cliped_weights.detach() + cliped_weights" (line 67 in the birealnet.py). 

If anyone understands the deeper meaning of these two lines of code, I would be very grateful for your replies.

**Comment by hadhe145** [comment]
Duplicate of #10
